### PR TITLE
Delete getContent function & Use Nullish coalescing operator

### PR DESCRIPTION
**Pull request by well-balanced**
Related issue: #31 

## Changes
- Remove unnecessary module `logMessageDate` which is being called on `src/index.js`
- Delete the `getContent` function
- Use Nullish coalescing operator

### CODE REVIEW

File: src/scripts/expletives.js [hunk 1]
```diff
@@ -63,7 +63,8 @@ const englishKoreanRegex = /([\w]+|[\x00-\x7F\x80-\xFF\u0100-\u017F\u0180-\u024F
 
 /* Checks if message has expletives */
 function check(message) {
-	let contentArray = getContent(message).split(englishKoreanRegex);
+  if (!message.content) return
```
Comment (Fox-Islam) [inline src/scripts/expletives.js]: This should have two more spaces to match the 4 space width indentation of the rest of the code

**Review by Fox-Islam — CHANGES_REQUESTED** [review]
Thanks for this! ✨ Just had one comment

File: src/scripts/expletives.js (hunk 1 above)
Comment (well-balanced) [reply src/scripts/expletives.js]: Oops I see that

File: src/scripts/expletives.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ const englishKoreanRegex = /([\w]+|[\x00-\x7F\x80-\xFF\u0100-\u017F\u0180-\u024F
 
 /* Checks if message has expletives */
 function check(message) {
-  if (!message.content) return
+	if (!message.content) return
```
Comment (well-balanced) [inline src/scripts/expletives.js:66]: @Fox-Islam I fixed it. thx :)

**Review by Fox-Islam — APPROVED** [review]

**Review by chris-mcdonald-dev — APPROVED** [review]
Fantastic, it looks great!
I appreciate you working on this ✨ :)

**Pull request merged**